Sources/CassandraClient/Statement.swift

```diff
@@ -91,6 +91,21 @@ extension CassandraClient {
             try checkResult { cass_statement_set_paging_size(self.rawPointer, pagingSize) }
         }
 
+        /// Sets the starting page of the returned paginated results.
+        ///
+        /// The paging state token can be obtained by the `pagingStateToken()`
+        /// function on `Rows`.
+        ///
+        /// - Warning: The paging state should not be exposed to or come from
+        /// untrusted environments. The paging state could be spoofed and
+        /// potentially used to gain access to other data.
+        func setPagingStateToken(_ pagingStateToken: PagingStateToken) throws {
+            try checkResult {
+                let bytes = pagingStateToken.token.map(CChar.init)
```

Comment:
The extra allocation here seems unnecessary: we can use `withUnsafeBytes` of `pagingStateToken` and pass it in directly.

Reply:
Resolved.